### DISCUSSION TITLE
Help with theming the plugin

### DISCUSSION THREAD

**Discussion by hankertrix** [ibhagwan/fzf-lua#1542] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/36cd33e5-aaab-4bc3-be73-af5b3f35aafd)

How would I go about changing the red line at the side to a `>`, like Telescope's one. For example, from https://github.com/ibhagwan/fzf-lua/discussions/1330:

![image](https://github.com/user-attachments/assets/97e6116e-e1d9-4303-8039-dd29cb532cea)

I also can't seem to find the highlight groups to theme file name, the line numbers of the file, as well as the characters matched in the live grep. I might be missing something, but I went through all the highlight groups provided by the plugin and couldn't find anything that changed these highlights.

**Comment by ibhagwan** [comment, marked as answer]
> How would I go about changing the red line at the side to a >

See man fzf, if I’m not mistaken that would be the “marker” flag, add `fzf_opts= { ['--marker'] = '>' }`

> I also can't seem to find the highlight groups to theme file name, the line numbers of the file, as well as the characters matched in the live grep. I might be missing something, but I went through all the highlight groups provided by the plugin and couldn't find anything that changed these highlights.

These colors are controlled by ripgrep, you can change your neovim terminal colors (fzf-lua runs fzf in a terminal buffer). You can also look into theming your terminal color in neovim (4 is red, etc).

**Reply by hankertrix** [reply]
Ahhhh I see. That would be quite difficult to theme using a theme plugin. The theme plugin already has its own terminal colours as well.

**Comment by ibhagwan** [comment]
Forgot to mention that if you use `formatter = 'path.dirname_first'` you can use `FzfLuaDirPart|FzfLuaFilePart` highlight groups to override the ripgrep output.

**Reply by hankertrix** [reply]
Thanks, I'll try this!
